packages/internal/generated-clients/Makefile

```diff
@@ -53,3 +53,28 @@ generate-blockchain-data-types:
     -o /app/src/blockchain-data \
     -c /app/config/blockchain-data.config.json \
     --additional-properties=stringEnums=true
+
+
+.PHONY: generate-mr-api-client-tmp-minting-api
```

Comment:
TODO remove these commands and regenerate using the published version of the OpenAPI spec once released